comm.cpp

```diff
@@ -320,17 +321,24 @@ int UdpEndpoint::open(const char *ip, unsigned long port)
     sockaddr.sin_addr.s_addr = inet_addr(ip);
     sockaddr.sin_port = htons(port);
 
-    if (setsockopt(fd, SOL_SOCKET, SO_BROADCAST, &broadcast_val, sizeof(broadcast_val))) {
-        log_error_errno(errno, "Error enabling broadcast in socket (%m)");
-        goto fail;
+    if (to_bind) {
```

Comment:
more on this: If a UDP socket is unbound and either sendto or connect are called on it, the system will automatically bind it for you and thus the recvfrom call later on will succeed.... so I still don't think this is needed.